hyperspy/drawing/image.py

```diff
@@ -93,6 +93,7 @@ def __init__(self, title="", **kwargs):
         self.figure = None
         self.ax = None
         self.title = title
+        self.mesh = None
```

Comment:
If we can avoid adding more attribute, it would be good to do so! Following the comment below, it should be possible?

Reply:
Removed it